src/mlpack/methods/ann/layer/randomized_relu_impl.hpp

```diff
@@ -0,0 +1,87 @@
+/**
+ * @file methods/ann/layer/randomized_relu_impl.hpp
+ * @author Shubham Agrawal
+ *
+ * @code
+ * @article{Bing Xu2015,
+ *  author = {Bing Xu, Naiyan Wang, Tianqi Chen, Mu Li},
+ *  title = {Empirical Evaluation of Rectified Activations in Convolutional 
+ *      Network},
+ *  year = {2015},
+ *  url = {https://arxiv.org/pdf/1505.00853.pdf}
+ * }
+ * @endcode
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_RRELU_IMPL_HPP
+#define MLPACK_METHODS_ANN_LAYER_RRELU_IMPL_HPP
+
+// In case it hasn't yet been included.
+#include "randomized_relu.hpp"
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template<typename InputDataType, typename OutputDataType>
+RReLU<InputDataType, OutputDataType>::RReLU(
+    const double lowerBound,
+    const double upperBound):
+    lowerBound(lowerBound),
+    upperBound(upperBound),
+    deterministic(false),
+    alpha(0.0)
+{
+  if (lowerBound < 1.0 || upperBound < 1.0)
```

Comment:
Hi ! I think we should also put a check to make sure that `lowerBound` is less than `upperBound`. 
So that we dont face issues when we pass them in math::Random as here 
```
alpha = 1.0 / math::Random(lowerBound, upperBound);
```